src/foam/nanos/controller/ApplicationController.js

```diff
@@ -510,23 +511,18 @@ foam.CLASS({
       });
     },
 
-    async function render() {
+    function render() {
       this.addMacroLayout();
       this.onClientLoad();
-      // if ( ! this.client || ! this.client.auth ) {
-      //   await this.reloadClient();
-      // }
-      this.fetchSubject();
```

Comment:
Believe we added this fetch subject so otLogin works properly

Reply:
it's not in the development ApplicationController.